controllers/users-controller.js

```diff
@@ -16,6 +18,33 @@ const { JWT_KEY } = require('../config');
  * @param {string} appReq.body.password - user provided password
  */
 exports.registerUser = (appReq, appRes) => {
+  
+   /**
+   *  Checks whether any of the fields are empty while submission.
+   *  Checks whether the email address is of a valid format
+   *  Checks whether password is of minimum 6 characters & that it has atleast one number & atleast one specail character. 
+   */
+  if(!appReq.body.email && !appReq.body.name && !appReq.body.password){
```

Comment:
Nearly there - this now is equivalent to all null; you don't want to use the `!` it means `not` for each user credential.

For example, if appReq.body.email = null (is empty and thus), `!appReg.body.email` = `true` and vice versa.